### PR TITLE
update integration after recent deprecations

### PR DESCRIPTION
**Pull request by AleksanderWWW**

### CODE REVIEW

File: src/neptune_detectron2/impl/__init__.py [hunk 1]
```diff
@@ -142,13 +148,13 @@ def _log_checkpoint(self, final: bool = False) -> None:
         checkpoint_path = self.trainer.checkpointer.get_checkpoint_file()
 
         with open(checkpoint_path, "rb") as fp:
-            self._run[neptune_model_path] = File.from_stream(fp)
+            self._root_object[neptune_model_path] = File.from_stream(fp)
```
Comment (kshitij12345) [inline src/neptune_detectron2/impl/__init__.py:151]: I think this should stay `self._run` and preserve the base namespace.

Comment (AleksanderWWW) [reply src/neptune_detectron2/impl/__init__.py:151]: Ok, makes sense

**Review by kshitij12345 — APPROVED** [review]
LGTM! Thank you

File: src/neptune_detectron2/impl/__init__.py [hunk 2]
```diff
@@ -56,7 +60,7 @@ class NeptuneHook(hooks.HookBase):
     """Hook implementation that sends the logs to Neptune.
 
     Args:
-        run: Pass a Neptune run object if you want to continue logging to an existing run.
+        run: Pass a Neptune run or handler object if you want to continue logging to an existing run.
```
Comment (normandy7) [inline src/neptune_detectron2/impl/__init__.py:63]:
```suggestion
        run: Pass a Neptune run or namespace handler object if you want to continue logging to an existing run.
```

**Review by normandy7 — APPROVED** [review]

**Pull request merged**